resources/swapi/character.yaml

```diff
@@ -0,0 +1,110 @@
+"$schema": https://json-schema.org/draft/2020-12/schema
+"$id": character
+title: Character
+type: object
+required:
+- gid
+- id
+- label
+uniqueKeys:
+- - gid
+- - label
+  - id
+links:
+- rel: homeworld
+  href: homeworld/{id}
+  templateRequired:
+  - id
+  targetSchema:
+    "$ref": homeworld
+  templatePointers: id
```

Comment:
(Pending resolution of how  has_one is represented in the data)
if "all references are arrays"
  either  /homeworld/-/id or /homeworld/0/id 
if "has_ones should be scalar"
  /homeworld/id